Sources/MapboxDirections/DirectionsOptions.swift

```diff
@@ -431,10 +431,15 @@ open class DirectionsOptions: Codable {
      The path of the request URL, not including the hostname or any parameters.
      */
     var path: String {
-        guard let coordinates = coordinates, !coordinates.isEmpty else {
+        guard let coordinates = coordinates else {
```

Comment:
By the way, why it's nil? I see that the `coordinates` property can never be `nil` and we can make it non-nil as it's internal.